diesel-wasm-sqlite/src/ffi.rs

```diff
@@ -82,11 +82,13 @@ pub fn raw_copy_to_sqlite<B: Into<Uint8Array>>(bytes: B, dst: *mut u8) {
 pub unsafe fn raw_copy_from_sqlite(src: *mut u8, len: u32, buf: &mut [u8]) {
     let wasm = crate::get_sqlite_unchecked().inner().wasm();
     let mem = wasm.heap8u();
-    let offset = (src as u32) / std::mem::size_of::<u8>() as u32;
-    // this is safe because we view the slice and immediately copy it into
-    // our memory.
-    let view = Uint8Array::new_with_byte_offset_and_length(&mem, offset, len);
-    view.raw_copy_to_ptr(buf.as_mut_ptr())
+    mem.slice(src as u32, src as u32 + len)
```

Comment:
this is interesting! I wonder why the other method didn't work. It seems to be doing the same/similar thing